docs/build/op_template.md

```diff
@@ -0,0 +1,42 @@
+# {{ data.name }}
+
+{% if 'subclass_of' in data -%}
+_subclass of [{{ data.subclass_of }}](./{{ data.subclass_of }}.md)_
+
+{% endif -%}{{ data.description }}
+
+{% if data.examples -%}
+### examples
+
+{% for example in data.examples -%}
+- [input](../examples/{{ example.input }}), [output](../examples/{{ example.output }})
+{% endfor %}
+{% endif %}### input requirements
```

Comment:
Was a newline intended here?

Reply:
I find whitespace management in jinja really annoying. It looks weird here, but I think this produces the output that I want.